sungrow/translations/250c9b83-1127-4013-bbd0-11e7ea482057-en_US.ts

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Probably not complete. Please run `make lupdate` in the build directory in order to update before pushing.